tests/routes_tests/cbauthentication_tests/cb_login.test.js

```diff
@@ -0,0 +1,35 @@
+const dbConnection = require('../../../react-backend/database/dbConnection.js');
+const test = require('tape');
+const request = require('supertest');
+const app = require('../../../react-backend/app');
+
+test('Route cblogin | post | successful request', (t) => {
+  t.plan(3);
+  const successPayload = { formEmail: 'findmyfroggy@frogfinders.com', formPswd: 'Funnyfingers11!' };
+  const failPayload = { formEmail: 'findmyfroggy@frogfinders.com', formPswd: 'password123' };
```

Comment:
If this test case is about a "successful request", why are we also posting payloads that return nominally unsuccessful responses?

Reply:
☝️ see my previous waffle...